ELK_Listener/ci_listener.yaml

```diff
@@ -220,7 +220,7 @@
 
                  # ISO8601 regex.  We use this for metrics timestamps
                  iso8601 = re.compile(r'(\d{{4}})-(0[1-9]|1[0-2])-(0[1-9]|1[0-9]|2[0-9]|3[01])(|[tT\s])(0[0-9]|1[0-9]|2[0-3]):(0[0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9]):(0[0-9]|1[0-9]|2[0-9]|3[0-9]|4[0-9]|5[0-9])Z')
-                 expected_Keys=['component', 'trigger', 'tests', 'job_link_back', 'base_distro', 'compose_id', 'brew_task_id', 'create_time', 'completion_time', 'CI_infra_failure', 'CI_infra_failure_desc', 'job_names', 'CI_tier', 'build_type', 'owner', 'content-length', 'destination', 'expires', 'xunit_links']
+                 expected_Keys=['component', 'trigger', 'tests', 'job_link_back', 'base_distro', 'compose_id', 'brew_task_id', 'create_time', 'completion_time', 'CI_infra_failure', 'CI_infra_failure_desc', 'job_names', 'CI_tier', 'build_type', 'owner', 'content-length', 'destination', 'expires', 'xunit_links', 'jenkins_build_url']
```

Comment:
We are changing 'job_link_back' to 'jenkins_job_url' as discussed in the doc.
Cc @jcanderle in case doc needs update accordingly too.